src/test/java/org/fisco/bcos/channel/client/TransactionResourceTest.java

```diff
@@ -0,0 +1,149 @@
+package org.fisco.bcos.channel.client;
+
+import java.math.BigInteger;
+import java.util.List;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.fisco.bcos.web3j.crypto.Credentials;
+import org.fisco.bcos.web3j.crypto.ECKeyPair;
+import org.fisco.bcos.web3j.crypto.Keys;
+import org.fisco.bcos.web3j.protocol.Web3j;
+import org.fisco.bcos.web3j.protocol.channel.ChannelEthereumService;
+import org.fisco.bcos.web3j.protocol.core.DefaultBlockParameter;
+import org.fisco.bcos.web3j.protocol.core.methods.response.BcosBlock;
+import org.fisco.bcos.web3j.protocol.core.methods.response.BcosTransaction;
+import org.fisco.bcos.web3j.protocol.core.methods.response.MerkleProofUnit;
+import org.fisco.bcos.web3j.protocol.core.methods.response.TransactionReceiptWithProof;
+import org.fisco.bcos.web3j.protocol.core.methods.response.TransactionWithProof;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class TransactionResourceTest {
+    static Logger logger = LoggerFactory.getLogger(TransactionResourceTest.class);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/fisco/web3sdk/pullRequest?prid=4840376)